libs/host/Configuration/Options.cs

```diff
@@ -224,6 +224,10 @@ internal sealed class Options
         [Option("cluster-tls-client-target-host", Required = false, HelpText = "Name for the client target host when using TLS connections in cluster mode.")]
         public string ClusterTlsClientTargetHost { get; set; }
 
+        [OptionValidation]
+        [Option("server-certificate-required", Required = false, HelpText = "Whether server TLS certificate is required by clients on the server side, e.g., for cluster gossip and replication.")]
```

Comment:
```suggestion
[Option("server-certificate-required", Required = false, HelpText = "Whether server TLS certificate is required by clients e.g., for cluster gossip and replication.")]
````

Reply:
It would be more precise to say "clients on the server side" else it is confusing how a server is asking for client information.

Reply:
```suggestion
[Option("server-certificate-required", Required = false, HelpText = "Whether server TLS certificate is required by clients established on the server side, e.g., for cluster gossip and replication.")]
```

Reply:
OK, please mirror the text change into defaults.conf as well